x-pack/plugins/stack_connectors/server/connector_types/sentinelone/index.ts

```diff
@@ -20,6 +25,8 @@ import { SentinelOneConfig, SentinelOneSecrets } from '../../../common/sentinelo
 import { SentinelOneConnector } from './sentinelone';
 import { renderParameterTemplates } from './render';
 
+const toApiTag = (tagName: string) => `api:${tagName}`;
```

Comment:
Would it make sense to centralize the code to add this `api:` prefix in the `ensureAuthorizedToExecute` function of the _action_executor.ts_ file, instead of the `getSubActionPrivileges` implementations?

I understand this prefix is something needed specifically for the `authorization.ensureAuthorized` call, right? And also this way we won't have to duplicate the same formatting in all connectors.

Reply:
I can do that if it makes sense, but it seemed like there was already code in place that retrieves "additional privileges" and just adds it to the `chackPrivileges()` call, so not sure what the impact there would be.

Reply:
btw - ideally we should be using the Security plugin `actions.api.get()` method, but we don't have access to it here.  Also, I'm not sure that `api` would be used in all instances - there are [other tags that kibana security supports](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security/server/authorization/actions/actions.ts#L31-L39) and IMO it should be the responsibility of the connector to determine what the tag should be. Still open to make the change - but wanted to point that out

Reply:
hmmm yeah that's right. We can not assume all privileges will be `api` privileges. 
Okay, this approach looks good then!